utils.py

```diff
@@ -47,6 +48,16 @@
 SECONDS_IN_HOUR = 60 * 60
 SECONDS_IN_MINUTE = 60
 
+T = TypeVar('T')
+U = TypeVar('U')
+DictListT = TypeVar('DictListT', Dict[Any, Any], List[Any])
+
+# Every use of constants is followed by # type: ignore[attr-defined] because
```

Comment:
Comments like these get hard to manage over time. There are 2 good practices that are followed so that their usage is minimised:
1. Create an issue stating removal of this comment and write it as a TODO(#issue_no) at all the applicable places. This new issue you create will state "remove these comments when its dependency has been met" (by referencing to the issue talking about solving the dependency) "and also state what further action should be taken".
2. Plan in such a way that dependencies are met first, and then further changes are made on top of it. This keep the process clean, understandable and save time in adding/deleting todo comments.